scripts/add.tokens.erc20.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+if [ -n "$1" ]; then
+  ETHERSCAN_API_KEY=$1
+  node ./scripts/add.tokens.erc20.mjs --etherscan-api-key "$ETHERSCAN_API_KEY"
+else
+  node ./scripts/add.tokens.erc20.mjs
+fi
+
+#npm run format && npm run lint
```

Comment:
You want to uncomment this? Or remove it its a leftover?

Reply:
good question: do you think i shall put it there for good measure? or in an automatic job we will put it in the job and not in the script?

Reply:
Maybe there that way when we run it in the CI but also locally, the output is formatted?

Reply:
actually, different change in the last commit: i included the optional run of `npm run format && npm run lint` only if files were created

otherwise it would have run everytime